content/docs/other/a_word_on_backward_compatibility.md

```diff
@@ -18,10 +18,13 @@ This is the current list of feature flags:
 
 | **Flag**                               | **Since** | **Description** |
 |----------------------------------------|-----------|-----------------|
+| no\_task\_chaining\_cmdline\_args      | 1.4       | |
+| task\_chaining\_cmdline\_args          | 1.4       | |
 | 1.3                                    | 1.3       | Activating the new template engine by default. |
 | no\_template\_ng                       | 1.3       | Disabling the new template engine. |
 | 1.0                                    | 1.0       | Disabling usage of a tty. This increases compatibility for remote execution. Furthermore, all features from earlier versions are activated. |
 | no\_autodie                            | 1.0       | Will disable autodie feature. |
+| tty                                    | 1.0       | Enable pty usage for ssh connections. (Default) |
```

Comment:
I see you ordered the feature flags by version. That sometimes make `{no_,}` pairs to be listed apart from each other, and I think both orders have pros and cons.

Let's order on version then, until we figure out a better way to present them.

Reply:
We can include the same table twice: once ordered by version and once by name (grouping the disabling functions with their enabling counter parts).